app/views/admin/logs/_show.slim

```diff
@@ -0,0 +1,49 @@
+.panel.log.expandable
+  .panel-heading
+    .row
+      .col-sm-10
+        b => log.date ? l(log.date) : ''
+        => t(:dash)
+        => Person.find(log.agent_id).name
```

Comment:
Should just be able to do `log.agent.name` or more properly `log.agent_name` using a delegator. You should never need to call `find` or etc. in a template. All the data you need should be setup by the controller or passed as a local variable into the partial.